packages/auth/next-auth.d.ts

```diff
@@ -9,11 +10,20 @@ import { type AdapterAccount as $AdapterAccount } from "next-auth/adapters";
  */
 
 declare module "next-auth" {
-  interface Session {
+  interface Session extends DefaultSession {
```

Comment:
This is included in `nextjs` app because it's a type file in our auth package?

Reply:
Yea I'm not quite sure why this augmentation doesn't work globally without including this file explicitly everywhere else. Have asked in turbo's discord without any answer. 

Cal.com also does this https://github.com/calcom/cal.com/blob/0c746ec10ffefa13deca60dcb7d002d2ecb7f2fd/packages/tsconfig/react-library.json#L11